core/src/main/res/values/strings.xml

```diff
@@ -901,4 +901,9 @@
     <string name="on_demand_config_setting_changed">Setting updated successfully.</string>
     <string name="on_demand_config_stream_text">Looks like you stream a lot. Do you want episode lists to show stream buttons?</string>
     <string name="on_demand_config_download_text">Looks like you download a lot. Do you want episode lists to show download buttons?</string>
+
+    <string name="KEYCODE_MEDIA_NEXT">87</string>
+    <string name="KEYCODE_MEDIA_PREVIOUS">88</string>
+    <string name="KEYCODE_MEDIA_REWIND">89</string>
+    <string name="KEYCODE_MEDIA_FAST_FORWARD">90</string>
```

Comment:
Not sure if this is right here, I can't make those integers because Android's `ListPreference` only works with strings.

Reply:
Oh, okay. I did not know that. Could you move them to their own xml file then, please? Something like `keycodes.xml` (Android does not care about the file names when defining strings). Also, please use lower case letters to be more consistent with other strings.